hack/test/cmd/basic-integration.go

```diff
@@ -0,0 +1,210 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package cmd
+
+import (
+	"context"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/spf13/cobra"
+	"github.com/talos-systems/talos/hack/test/internal/pkg/constants"
+	"github.com/talos-systems/talos/hack/test/internal/pkg/runner"
+)
+
+var cleanup bool
+var talosImage = "docker.io/autonomy/talos:"
+
+// Add basic-integration command
+var bi = &cobra.Command{
```

Comment:
The naming convention used by cobra would call for this to be `basicIntegrationCmd`. Was this created with `cobra add`?

Reply:
nah, I just threw this together manually. but I'll rename for sure.